src/osip-simulation-controller/src/main/resources/Bundle.properties

```diff
@@ -14,3 +14,17 @@ opcua.pipe.flowRate=Flow rate
 opcua.motor=Motor
 opcua.motor.rpm=Rotations per minute
 opcua.pipe.in=Incoming pipe
+controller.helpdialog.title=Help
+monitoring.helpdialog.closeButton=Close
+controller.helpdialog.mainTab.header=General
+controller.helpdialog.mainTab.introduction=The OSIP production site simulation simulates a production site consisting of three upper tanks containing liquids, which get mixed through some pipes into a mix tank. It is possible to adjust a motor speed, temperatues and valves.
```

Comment:
Komma im ersten Satz ist unnötig und ich hätte geschrieben "liquids which get filled [..]", da das ansonsten impliziert, dass die Flüssigkeiten in den Rohren gemischt werden.